Sources/BraintreeCore/BTShopperInsightsRequest.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+/// Buyer data required to use the Shopper Insights feature.
+/// - Note: This feature is in beta. It's public API may change in future releases.
+@objcMembers public class BTShopperInsightsRequest: NSObject {
+    
+    /// The buyer's email address.
+    public let email: String
+    
+    /// The buyer's country code prefix to the national telephone number. An identifier for a specific country.
+    public let phoneCountryCode: String
+    
+    /// The buyer's national phone number.
+    public let phoneNationalNumber: String
```

Comment:
Good question - since we can't test the actual API yet, I'm not sure how it will treat special characters. It _should_ be able to handle the case with characters + digits fine, as well as only digits. The examples they've given us show only as digits.
 
We can document it as only numbers now to be safe

Reply:
Done in 5973db06a76bc98e8dd013a4630ba5a58c3bbd26!